synapse/models/risk.py

```diff
@@ -185,23 +248,36 @@ def getModelDefs(self):
                     ('campaign', ('ou:campaign', {}), {
                         'doc': 'Set if the attack was part of a larger campaign.',
                     }),
+                    ('compromise', ('risk:compromise', {}), {
+                        'doc': 'A compromise that this attack contributed to.',
+                    }),
                     ('prev', ('risk:attack', {}), {
                         'doc': 'The previous/parent attack in a list or hierarchy.',
                     }),
                     ('actor:org', ('ou:org', {}), {
-                        'doc': 'The org that carried out the attack.',
+                        'deprecated': True,
```

Comment:
So we have some very nice deprecation docs that would love to have these added to them :smiley: